tendermint/src/lib.rs

```diff
@@ -4,6 +4,7 @@
 //! blockchain networks, including chain information types, secret connections,
 //! and remote procedure calls (JSONRPC).
 
+#![cfg_attr(docsrs, feature(doc_cfg))]
```

Comment:
I wired up `doc_cfg`, which allows you to document which Cargo features need to be enabled to use certain types/methods in rustdoc.

Rendered example:

<img width="575" alt="Screen Shot 2020-08-12 at 8 26 45 PM" src="https://user-images.githubusercontent.com/37432020/90090970-2e8a0580-dcda-11ea-8d46-fcb6d2d4a105.png">